src/main.ts

```diff
@@ -17,7 +17,8 @@ const v1 = express.Router();
 
 v1.route("/event")
   .get(read_all_events)
-  .post(create_event);
+  .post(create_event)
+  .post(update_one_event)
```

Comment:
I assume we probably don't want two POST paths. Maybe use PUT instead of POST for the update.

Reply:
Yes, we want to use put instead of post here.